From a6bb3358ddae36212484fa3e0130884462d631dd Mon Sep 17 00:00:00 2001
From: Rogerio Alves <rogealve@br.ibm.com>
Date: Fri, 13 Oct 2017 10:56:38 -0300
Subject: [PATCH] Deleting Make from release version.

libexpat tar release on github uses a Makefile to generate man file.
This is generated by docbook2X package which may not avaliable in old
distros like RHEL 6. Since we should not update the man for this release
the Makefile can be deleted.
---
 expat/doc/Makefile | 24 ------------------------
 1 file changed, 24 deletions(-)
 delete mode 100644 expat/doc/Makefile

diff --git a/expat/doc/Makefile b/expat/doc/Makefile
deleted file mode 100644
index d8567980..00000000
--- a/expat/doc/Makefile
+++ /dev/null
@@ -1,24 +0,0 @@
-# Copyright (C) Sebastian Pipping <sebastian@pipping.org>
-# Licensed under MIT/X License
-
-DOCBOOK_TO_MAN = docbook2x-man
-SGML_TO_XML = sgml2xml-isoent
-
-.PHONY: all
-all: xmlwf.1
-
-xmlwf.1: XMLWF.1
-	cp $< $@
-
-XMLWF.1: xmlwf.xml
-	which $(DOCBOOK_TO_MAN)
-	$(DOCBOOK_TO_MAN) $< > $@
-
-xmlwf.xml: xmlwf.sgml
-	which $(SGML_TO_XML)
-	$(SGML_TO_XML) $< > $@ || true
-	grep -q '^<?xml' $@
-
-.PHONY: clean
-clean:
-	$(RM) xmlwf.1 XMLWF.1 xmlwf.xml
